book/website/community-handbook/onboarding-offboarding/offboarding-core-team.md

```diff
@@ -0,0 +1,53 @@
+# 🌙 We are sad to see you leave *The Turing Way* Core Team! 🌙
+
+:::info
+Whether it's changed responsibilities, a new job, shifting interests, or any other reason –– we understand that life happens, and that communities will always be shifting and evolving! While we are sad to see you go for now, we hope that you'll stay engaged with us in *The Turing Way* community! 
+
+This offboarding document is a living document – as is *The Turing Way* itself. As you'll learn from being in this community: this is a constantly-evolving project that holds [emergence as a core value](https://hackmd.io/kVkwKSYuSLWWhLzlNSuT0g). In other words: as the project evolves – so will the team that enables its sustainability as a project.
+:::
+
+# TTW Core Team Onboarding checklists
+
+## Accounts and Permissions
+- [ ] Remove from #ttw-core-team channel on Slack
+- [ ] Change [TTW Github repository](https://github.com/alan-turing-institute/the-turing-way) permissions as needed
+- [ ] Add to 'Alumni' section of [Ways of Working](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) page on TTW Github repository 
+- [ ] Remove from [Core Team Google group](https://groups.google.com/u/1/g/core-team-2022) (currently dormant)
+- [ ] (If applicable) Remove from HackMD team for The Turing Way
+- [ ] (If applicable) Give Accounts/Password Access on Keybase
+- [ ] (If applicable) Remove from [The Turing Way Core Google Drive](https://drive.google.com/drive/u/0/folders/1bdjA-fBotMi9CnjDqu84d5hPZWwjgi-x)
+    - [ ] Remove from [promotion pack](https://drive.google.com/drive/u/1/folders/1mzGmbJkPnP5q1goQesxDc_E5zAPL0eTF) folder
+    - [ ] Remove from [conference talks/workshop applications](https://drive.google.com/drive/u/0/folders/1eNHq9sXq6Z8Q5Ep0MzNS0GsB3FKgTi3E) folder
+- [ ] (If applicable) Remove from Canva team
+- [ ] (If applicable) Remove from Penpot team
+- [ ] (If applicable) Remove from Keybase group
+
+## Personal information
+- [ ] Move information on [TTW Team slides](https://docs.google.com/presentation/d/1PeQ_gg5tOJVhOA4aSnVumsowD-bfRFEroWfGsxxaGCw/edit#slide=id.g1aee8e18598_0_0) to 'Alumni' section
+- [ ] (If applicable) remove [TTW project page](https://www.turing.ac.uk/research/research-projects/turing-way) on Alan Turing website
+- [ ] (If applicable) remove yourself to [TPS/Whitaker Lab slide deck](https://docs.google.com/presentation/d/1g-plncmi00FK97QuYDfqq7oPzUmL0xkHm1hy4X81IX8/edit#slide=id.gf5e62e1b31_1_0)
+
+## Events & Calendars
+- [ ] Remove from TTW Community Calendars
+    - [ ] The Turing Way Community (public-facing)
+    - [ ] The Turing Way Core Team (internal)
+- [ ] Remove from calendar invites for The Turing Way core team
+    - [ ] Coworking Calls
+    - [ ] Collaboration Cafes
+    - [ ] Fireside Chat
+    - [ ] Community Chats
```

Comment:
I would add a brief explanation about how to remove these calendar invitations from the calendar.